sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <full-page-container layout-template="centered-card-tab">
+    <template #header>
+      <header-navigator
+        title="Assessments"
+        :routeLocation="goBackRouteParams"
+        subTitle="View Request"
+      >
+        <template
+          #buttons
+          v-if="
+            applicationOfferingChangeRequestDetails.status ===
+            ApplicationOfferingChangeRequestStatus.InProgressWithSABC
+          "
+        >
+          <v-row class="p-0 m-0">
+            <v-btn color="primary" variant="outlined"
+              >Decline reassessment</v-btn
+            >
+            <v-btn class="ml-2" color="primary">Approve reassessment</v-btn>
+          </v-row>
+        </template>
+      </header-navigator>
+      <application-offering-change-details-header
+        class="m-4"
+        :headerDetails="headerDetails"
+      />
+    </template>
+    <template
+      #alerts
+      v-if="
+        applicationOfferingChangeRequestDetails.status ===
+        ApplicationOfferingChangeRequestStatus.InProgressWithStudent
+      "
+      ><application-offering-change-request-banner />
+    </template>
+    <template #tab-header>
+      <student-application-offering-change-details
+        class="mb-6"
+        :studentName="applicationOfferingChangeRequestDetails.studentFullName"
+        :applicationNumber="
+          applicationOfferingChangeRequestDetails.applicationNumber
+        "
+        :locationName="applicationOfferingChangeRequestDetails.locationName"
+        :reasonForChange="applicationOfferingChangeRequestDetails.reason"
+        :accessedNoteDescription="
+          applicationOfferingChangeRequestDetails.assessedNoteDescription
+        "
+        :applicationOfferingChangeRequestStatus="
+          applicationOfferingChangeRequestDetails.status
+        "
+      />
+      <v-tabs stacked v-model="tab" color="primary">
+        <v-tab value="requested-change" :ripple="false">Requested Change</v-tab>
+        <v-tab value="active-offering" :ripple="false">Active Offering</v-tab>
```

Comment:
wasn't the suggestion to use router tabs?

Reply:
I think it was discussed as nice to have particularly for this refactoring.

Reply:
Not planning on doing it in this PR as it will further extend the PR.